tests/suites/test_suite_hmac_drbg.misc.data

```diff
@@ -10,6 +10,10 @@ HMAC_DRBG entropy usage SHA-256
 depends_on:MBEDTLS_SHA256_C
 hmac_drbg_entropy_usage:MBEDTLS_MD_SHA256
 
+HMAC_DRBG entropy usage SHA-256 accumulator
+depends_on:MBEDTLS_SHA256_C:!MBEDTLS_SHA512_C
```

Comment:
Nowadays the entropy accumulator also uses SHA-256 if `MBEDTLS_ENTROPY_FORCE_SHA256` is set.